depends: Fix `make --just-print` command

hebasto opened the pull request:
Dry run `make` is very useful for debugging. From the [GNU make manual](https://www.gnu.org/software/make/manual/html_node/Instead-of-Execution.html#Instead-of-Execution):
> ‘-n’
> ‘--just-print’
> ‘--dry-run’
> ‘--recon’
>
>    “No-op”. Causes `make` to print the recipes that are needed to make the targets up to date, but not actually execute them. Note that some recipes are still executed, even with this flag (see How the MAKE Variable Works). Also any recipes needed to update included makefiles are still executed (see How Makefiles Are Remade).

On master (03689317021a72431762c1974530f2a980a7fffa) this option is not so useful:
```
$ make -n -C depends zlib
make: Entering directory '/home/hebasto/guix/GitHub/bitcoin/depends'
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps /home/hebasto/guix/GitHub/bitcoin/depends/sources
rm -f /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash
touch /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash
cd /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps; ( test -f /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz || (     ( mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 && echo Fetching zlib-1.2.11.tar.gz from https://www.zlib.net && curl --location --fail --connect-timeout 30 --retry 3 -o "/home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" "https://www.zlib.net/zlib-1.2.11.tar.gz" && echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" > /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && sha256sum -c /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && mv /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz && rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 ) ||     ( mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 && echo Fetching zlib-1.2.11.tar.gz from https://bitcoincore.org/depends-sources && curl --location --fail --connect-timeout 30 --retry 3 -o "/home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" "https://bitcoincore.org/depends-sources/zlib-1.2.11.tar.gz" && echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" > /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && sha256sum -c /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && mv /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz && rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 )))
cd /home/hebasto/guix/GitHub/bitcoin/depends/sources; sha256sum zlib-1.2.11.tar.gz >> /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash;
touch /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash
echo Extracting zlib...
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58; mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58 && echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz" > /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.zlib-1.2.11.tar.gz.hash &&  sha256sum -c /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.zlib-1.2.11.tar.gz.hash && tar --no-same-owner --strip-components=1 -xf /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.stamp_extracted
echo Preprocessing zlib...
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58 /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.patches-4416f3f0c58
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58; 
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.stamp_preprocessed
echo Configuring zlib...
rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu; mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/lib; cd /home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu; 
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.;     PKG_CONFIG_LIBDIR=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/lib/pkgconfig PKG_CONFIG_PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/share/pkgconfig CMAKE_MODULE_PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/lib/cmake PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/native/bin:/home/hebasto/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin   env CC="gcc -m64" CFLAGS="-pipe -O2 -I/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/include -fPIC" RANLIB="ranlib" AR="ar"        ./configure --static --prefix=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu
/bin/sh: 1: cd: can't cd to /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/.
env: ‘./configure’: No such file or directory
make: *** [funcs.mk:268: /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-4416f3f0c58/./.stamp_configured] Error 127
```

With this PR:
```
$ make -n -C depends zlib
make: Entering directory '/home/hebasto/guix/GitHub/bitcoin/depends'
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps /home/hebasto/guix/GitHub/bitcoin/depends/sources
rm -f /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash
touch /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash
cd /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps; ( test -f /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz || (     ( mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 && echo Fetching zlib-1.2.11.tar.gz from https://www.zlib.net && curl --location --fail --connect-timeout 30 --retry 3 -o "/home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" "https://www.zlib.net/zlib-1.2.11.tar.gz" && echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" > /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && sha256sum -c /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && mv /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz && rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 ) ||     ( mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 && echo Fetching zlib-1.2.11.tar.gz from https://bitcoincore.org/depends-sources && curl --location --fail --connect-timeout 30 --retry 3 -o "/home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" "https://bitcoincore.org/depends-sources/zlib-1.2.11.tar.gz" && echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp" > /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && sha256sum -c /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/.zlib-1.2.11.tar.gz.hash && mv /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz && rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11 )))
cd /home/hebasto/guix/GitHub/bitcoin/depends/sources; sha256sum zlib-1.2.11.tar.gz >> /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash;
touch /home/hebasto/guix/GitHub/bitcoin/depends/sources/download-stamps/.stamp_fetched-zlib-zlib-1.2.11.tar.gz.hash
echo Extracting zlib...
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a; mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a && echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz" > /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.zlib-1.2.11.tar.gz.hash &&  sha256sum -c /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.zlib-1.2.11.tar.gz.hash && tar --no-same-owner --strip-components=1 -xf /home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.stamp_extracted
echo Preprocessing zlib...
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.patches-38eb814ef0a
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a; 
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.stamp_preprocessed
echo Configuring zlib...
rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu; mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/lib; cd /home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu; 
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.;     PKG_CONFIG_LIBDIR=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/lib/pkgconfig PKG_CONFIG_PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/share/pkgconfig CMAKE_MODULE_PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/lib/cmake PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/native/bin:/home/hebasto/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin   env CC="gcc -m64" CFLAGS="-pipe -O2 -I/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/include -fPIC" RANLIB="ranlib" AR="ar"        ./configure --static --prefix=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/./.stamp_configured
echo Building zlib...
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.; PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/native/bin:/home/hebasto/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin   make libz.a
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/./.stamp_built
echo Staging zlib...
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a//home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.; PATH=/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/native/bin:/home/hebasto/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin   make DESTDIR=/home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a install
rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/.stamp_staged
echo Postprocessing zlib...
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu; 
touch /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu/.stamp_postprocessed
echo Caching zlib...
cd /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a//home/hebasto/guix/GitHub/bitcoin/depends/x86_64-pc-linux-gnu; find . | sort | tar --no-recursion -czf /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/zlib-1.2.11-38eb814ef0a.tar.gz -T -
mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-pc-linux-gnu/zlib
rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-pc-linux-gnu/zlib && mkdir -p /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-pc-linux-gnu/zlib
mv /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a/zlib-1.2.11-38eb814ef0a.tar.gz /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-pc-linux-gnu/zlib/zlib-1.2.11-38eb814ef0a.tar.gz
rm -rf /home/hebasto/guix/GitHub/bitcoin/depends/work/staging/x86_64-pc-linux-gnu/zlib/1.2.11-38eb814ef0a
cd /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-pc-linux-gnu/zlib; sha256sum zlib-1.2.11-38eb814ef0a.tar.gz > /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-pc-linux-gnu/zlib/zlib-1.2.11-38eb814ef0a.tar.gz.hash
make: Leaving directory '/home/hebasto/guix/GitHub/bitcoin/depends'
```

hebasto committed:
depends: Fix `make --just-print` command

laanwj commented:
I don't know enough about make syntax to review this but concept ACK, TIL that this is possible at all it seems useful for troubleshooting in some cases.

practicalswift commented:
Concept ACK

madietlx commented on depends/funcs.mk:
```diff
@@ -194,13 +194,13 @@ $($(1)_preprocessed): | $($(1)_extracted)
 	$(AT)rm -rf $(host_prefix); mkdir -p $(host_prefix)/lib; cd $(host_prefix); $(foreach package,$($(1)_all_dependencies), tar --no-same-owner -xf $($(package)_cached); )
-	$(AT)mkdir -p $$(@D)
-	$(AT)+cd $$(@D); $($(1)_config_env) $(call $(1)_config_cmds, $(1))
+	$(AT)+mkdir -p $$(@D)
```
This seems to be the only `+` left after your suggested changes (which look good to me, fixing `cd`ing into directories that have not been created). Wouldn't it be possible/better to drop this last `+`, too? Or is there a reasoning why this line has to be prefixed with a `+`, creating a folder?

To be honest: I've not run `make` to test it out and I'm new to the code base, so just asking.

hebasto replied:
@madietlx

> Or is there a reasoning why this line has to be prefixed with a `+`, creating a folder?

Not-creating that folder causes an error when one runs `make --dry-run`. The `+` forces `make` to execute a line even in `--dry-run` mode.

madietlx replied:
> Not-creating that folder causes an error when one runs `make --dry-run`. The `+` forces `make` to execute a line even in `--dry-run` mode.

@hebasto And that error would still occur - even though you did remove the `+` in front of the following `cd` line which caused the error in the first place?

hebasto replied:
Thanks. [Updated](https://github.com/bitcoin/bitcoin/pull/19891#issuecomment-691027367).

hebasto commented:
Updated ce3dc08ce1c9f5a977eea2d20a9cc3e352cd75bd -> 85e2b50888a598fccd771db1e5726159af147ff8 ([pr19891.01](https://github.com/hebasto/bitcoin/commits/pr19891.01) -> [pr19891.02](https://github.com/hebasto/bitcoin/commits/pr19891.02), [diff](https://github.com/hebasto/bitcoin/compare/pr19891.01..pr19891.02)):

- dropped unneeded change

madietlx reviewed (approved):
Code Review ACK 85e2b50888a598fccd771db1e5726159af147ff8

I've not run `make` to test the changes, but the code looks good to me.

hebasto commented:
Closing as this is a wrong approach due to https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html#MAKE-Variable

madietlx commented:
> Closing as this is a wrong approach due to https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html#MAKE-Variable

Could you explain why you think that your approach is wrong?

hebasto commented:
> Could you explain why you think that your approach is wrong?

It prevents passing `-j` to sub-`make`s.

madietlx commented:
> It prevents passing `-j` to sub-`make`s.

@hebasto I have to admit that I did not see this coming. I definitely need to set up a local dev environment to test those things out.

Thanks for your quick responses to my questions!
